rbac/server/swagger/proposals/proposals-id.yaml

```diff
@@ -0,0 +1,83 @@
+parameters:
```

Comment:
Repeat the file grouping for all endpoints. eg: One file would include

/proposals
/proposals/{proposal-id}

If it rolls up to the same url segment at the end, it should be in the same file